assets/scripts/trigger_algolia.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+which npm
```

Comment:
NIT: if it isn't, it could die if these weren't present and save you some CPU time if something ever goes awry this early on.

Reply:
I could actually get rid of the which statements. This is only called in an azure pipeline stage that is right after the "setup node" part